README.md

```diff
@@ -85,8 +85,6 @@ sudo /usr/local/share/gcm-core/uninstall.sh
 
 ---
 
-<!-- this explicit anchor should stay stable so that external docs can link here -->
-<a name="linux-install-instructions"></a>
```

Comment:
Is this actually linked from external docs?

If so, is there a commitment not to break this specific link for some reason?

Docs (anywhere) can link to headings in markdown using the `url#heading` syntax, as in `docs/wsl.md` in this commit. There isn't usually any need to add additional, explicit anchors for markdown headings.

Reply:
It's linked from at least the [GitHub docs](https://docs.github.com/en/get-started/getting-started-with-git/caching-your-github-credentials-in-git#git-credential-manager) on the topic (you may have to click over to the Linux tab) and possibly other vendors as well.

Reply:
Thanks! I'll experiment with a few options for solving this in a lint-friendly way then.

Are these docs rendered only on GitHub or also I  other locations that I should test the results against?

Reply:
If you mean the docs in this repo, no – we don't render them anyplace else.

Reply:
Turns out GitHub Flavored Markdown does not support custom heading links in any form.
So I've ended up simply adding an "ignore"  for the specific linting rule for this anchor (in 8cf643a)